student/views/registration.py

```diff
@@ -83,7 +97,8 @@ def get(self, request, registration_id):
                 'payable_line_item_amount_in_dollars': settings.CODING_BASICS_REGISTRATION_FEE_SGD,
```

Comment:
might wanna take a look at what this variable was used for. if there are no more usages in the rest of the codebase after you've removed it, best practice is to clean this up too! can clarify over a call if needed.